viper.go

```diff
@@ -1123,6 +1123,12 @@ func (v *Viper) SetEnvKeyReplacer(r *strings.Replacer) {
 	v.envKeyReplacer = r
 }
 
+// Returns the alias map, used for documentation generation.
+func GetAliases() map[string]string { return v.GetAliases() }
```

Comment:
Nit: Space between functions.

Also the both needs comments that start with "GetAliases".

Reply:
Normally I would agree, but I'm trying to write this in keeping with the existing style, which comments only the module level functions, and leaves no space between the module function and the instance implementation.  I will add the GetAliases prefix to the existing comment.